app/forms/collection_rights_form.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# This represents the form in the modal on the item page that appears when you
+# click "Set rights" in the sidebar
+class CollectionRightsForm < AccessForm
+  # @param [Cocina::Models::Collection] model
+  def initialize(model, default_rights: nil)
+    super
+  end
+
+  # TODO: validate the rights that is set doesn't include CDL
+
+  def sync
+    access_additions = CocinaAccess.from_form_value(rights)
+    updated_access = model.access.new(access_additions.value!)
+
+    @model = model.new(access: updated_access)
+  end
+
+  # all the rights except CDL
+  def rights_list
+    super.filter { |i| i.last != 'cdl-stanford-nd' }
```

Comment:
What is the significance of `last` here? It's looping over an array of arrays, and the last value in the inner array is where there's a label that could match?

Reply:
yes.